### ISSUE TITLE
How to generate the embeddings?

### ISSUE THREAD

**Issue by leechuck** [bio-ontology-research-group/D4#3]
Can you please add the description of how to generate the embeddings? Where to find the data sources and ontologies, how to run OPA2Vec, etc.?

**Comment by adeebnoor** [comment]
Hey Robert, eveything is written on the paper, do you still want me to add them here? @leechuck

**Comment by leechuck** [comment]
Yes, please make the embeddings available (ideally: put them on Zenodo or similar so we get a DOI).

**Comment by adeebnoor** [comment]
Embeddings are available under the following: https://bio2vec.cbrc.kaust.edu.sa/data/D4/embeddings/. 
Also, the paper is ready for you, I have fixed all comments. @leechuck
